### PR TITLE
Add Akka to the post-CurryOn spree next week

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Comment by heathermiller** [comment]
Woohoo! 🎉

**Comment by heathermiller** [comment]
Also a thought... @Philippus, would you be up for adding yourself as a maintainer for scala-lang? 😊

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Here is a list of projects that you could contribute to during the spree:
 
 | Project      			                                                            | Contact             |
 | ----------------------------------------------------------------------------------------- | ------------------- |
-| [ExampleProjectName] - Project Description         | [@maintainer1] |
+| [Akka] - toolkit for concurrent, distributed, and resilient message-driven applications | [@raboof] |
```
Comment (dwijnand) [inline README.md]: raboof needs a reference

Comment (raboof) [reply README.md]: woops, I sure do :laughing:

**Comment by raboof** [comment]
Is there anything we need to take care of before merging?

**Comment by Philippus** [comment]
@heathermiller sure, I can help out with scala-lang and docs.scala-lang

**Comment by SethTisue** [comment]
(we can always tweak further if needed)

**Pull request merged**